spec/requests/posts_spec.rb

```diff
@@ -12,9 +12,9 @@
 
   describe 'posts#show' do
     it 'returns a user posts!' do
-      get '/users/14/posts/3'
+      get '/users/1/posts/6'
       expect(response).to have_http_status(:ok)
-      expect(response.body).to include('User Post')
+      expect(response.body).to include('This is my second post')
       expect(response).to render_template(:show)
     end
```

Comment:
- [ ] This test is failing. Kindly fix them. You may need to create the user and post and use their IDs instead of using an arbitrary number as id: `@user.id` and `@post.id`.

  ![image](https://user-images.githubusercontent.com/54848363/229853641-a8984b97-bc04-4b7b-8874-a4b4d31a9c9c.png)